src/actions/vstudio/_vstudio.lua

```diff
@@ -352,6 +352,18 @@
 		return links
 	end
 
+---
+-- Assemble the list of library directories.
+--
+-- @param cfg
+--    The active configuration.
+-- @return
+--    The list of library directories, ready to be used in Visual Studio's
+--    AdditionalLibraryDirectories element.
+---
+	function vstudio.getLibraryDirectories(cfg)
+		return table.filterempty(config.getlinks(cfg, "system", "directory"))
+	end
```

Comment:
oh, and tthis doesn't turn out to be correct... vstudio.path calls "project.getrelative" where config.getlinks already returns paths that are relative...